src/blackmetal/launch/blackmetal.launch.py

```diff
@@ -22,12 +22,11 @@ def generate_launch_description():
 			parameters = [config_blackmetal],
 			emulate_tty=True
 		),
-		Node(
-			package='blackmetal',
-			executable='bm_logger',
-			output='screen',
-			parameters = [config_log],
-			emulate_tty=True
-		)
+		# Node(
+		# 	package='blackmetal',
+		# 	executable='log_server',
+		# 	output='screen',
+		# 	emulate_tty=True
+		# )
```

Comment:
preco zakomentovane?

Reply:
Hej to som zabudol odkomentovat ale je to v komite DO-NOT-MERGE

Reply:
odkomentovane